test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatConfigTests.cs

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using NuGet.CommandLine.XPlat;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace NuGet.XPlat.FuncTest
+{
+    [Collection("NuGet XPlat Config Test Collection")]
+    public class XPlatConfigTests
+    {
+
+        [Fact]
+        public void ConfigPathsCommand_ListConfigPathsWithArgs_Success()
+        {
+            // Arrange
+            using (var testInfo = new TestInfo("NuGet.Config"))
```

Comment:
style: It may look neater if you use a simple using statement: https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0063.